runtime/compiler/runtime/JITServerAOTDeserializer.cpp

```diff
@@ -388,6 +388,14 @@ JITServerAOTDeserializer::cacheRecord(const ClassLoaderSerializationRecord *reco
          );
       return false;
       }
+   else if (!chain)
+      {
+      if (TR::Options::getVerboseOption(TR_VerboseJITServer))
+         TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer,
+            "ERROR: Found class loader but not chain for first loaded class %.*s", RECORD_NAME(record)
```

Comment:
Isn't this a normal case when SCC is not used? If so, we should not be generating error messages.

Reply:
Yes, once the SCC isn't required we won't be looking up chains at all, so this message will be removed then. Right now the deserializer must be able to get the class chain (really the class chain offset) associated to a class loader in order to function properly, so to make this a standalone PR I had to add the extra check.